docs/notebooks/structure_function_showcase.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        id = np.array([[1],[2]]),
</pre><p>id is a native python function, we should try to avoid overwriting the namespace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Changed this to `lc_id`.